Meta/Lagom/Fuzzers/FuzzCyrillicDecoder.cpp

```diff
@@ -11,8 +11,8 @@
 
 extern "C" int LLVMFuzzerTestOneInput(uint8_t const* data, size_t size)
 {
-    auto* decoder = TextCodec::decoder_for("windows-1251"sv);
-    VERIFY(decoder);
-    decoder->to_utf8({ data, size });
+    auto decoder = TextCodec::decoder_for("windows-1251"sv);
+    VERIFY(decoder.has_value());
+    MUST(decoder->to_utf8({ data, size }));
```

Comment:
Doesn't MUST here mean the fuzzer will now find invalid utf-8? That doesn't seem terribly useful 😅

(Also below in the other fuzzers)

Reply:
That is a very good point! Thanks.